addons/utm/models/utm.py

```diff
@@ -42,14 +49,88 @@ def _group_expand_stage_ids(self, stages, domain, order):
         stage_ids = stages._search([], order=order, access_rights_uid=SUPERUSER_ID)
         return stages.browse(stage_ids)
 
+    def _merge_utm_campaigns(self, campaign=False, name=False, user_id=False, stage_id=False, tag_ids=False):
+        if len(self.ids) <= 1:
+            raise UserError(_('Please select more than one campaign from the list.'))
+        if not all(r.active for r in self):
```

Comment:
We try to avoid one letter variable, even in lambdas.
Here you could use "campaign".